### DISCUSSION TITLE
Connect to Codespace: Get codespace request failed with status 401. Correlation ID: FD58:0651:50342A:51D656:63D58467

### DISCUSSION THREAD

**Discussion by makspovarchuk** [community/community#45655] [Codespaces]
### Select Topic Area

Bug

### Body

help please i dont know what to do

**Comment by jge162** [comment]
Hello @makspovarchuk there is an issue with the authentication or authorization of the request.

clear browser cache and cookies.
Try to access Codespace from different browser or device.
Make sure you are in the correct Azure DevOps organization and project.

**Reply by makspovarchuk** [reply]
checked and tried it all, unfortunately, did not help, this is codespace cs50, in the browser I can open , but in the version vs code on pc I can not

**Reply by jge162** [reply]
Try to sign out and sign in again from the extension to refresh the token and make sure it is valid.

Check if the extension is enabled and properly configured in VS Code. You can do this by going to the Extensions view and looking for "Azure DevOps" in the list of installed extensions. If it's not enabled, click the "Enable" button.

**Reply by makspovarchuk** [reply]
I have problems with the GitHub Codespace extension I did not install Azure Devops, for some reason it does not want to work

**Reply by jge162** [reply]
here is a thorough guide. just double check you did all these steps to [add Codespaces to VS code](https://docs.github.com/en/codespaces/developing-in-codespaces/using-github-codespaces-in-visual-studio-code)

**Reply by jge162** [reply]
this will fix issue if you do step by step

**Reply by makspovarchuk** [reply]
thanks a lot i will try

**Reply by jge162** [reply]
No problem. Let me know how it goes.

**Reply by makspovarchuk** [reply]
this worked for me. You will need to open the Credential Manager in Windows and delete (or edit) the Github account there. thank you for trying to help
